cogs/highlights.py

```diff
@@ -116,40 +144,53 @@ async def highlightlist(self, ctx: commands.Context) -> None:
             )
             await ctx.send(embed=embed)
         else:
-            await ctx.send(":x: **You don't have any Highlights.**")
+            await ctx.send(f":x: **You don't have any Highlights{self._server_text(server, ' ', '')}.**")
+
+    def _hightlight_table_str(self, highlights: tuple[tuple[str, int], ...], ctx: commands.Context) -> str:
+        hl_table = []
+        max_pattern_len = len("pattern")
+        for pattern, gid in highlights:
+            max_pattern_len = max(max_pattern_len, len(pattern))
+            gid = f"{gid} : {ctx.bot.get_guild(gid).name}" if gid else "0"
+            hl_table.append((pattern.replace('`', ''), gid))
+        header = f"*`{'pattern':<{max_pattern_len}}  server`*\n"  # aligned italic header
+        return header + "\n".join(f"`{pattern:<{max_pattern_len}}  {gid}`" for pattern, gid in hl_table)
```

Comment:
For better readability, you should write variable names in full, not abbreviated (`highlight_table`, `max_pattern_length`, `guild_id`)

Reply:
Obsolete: replaced by guild list:

global `(0)`:
> `global-pattern`

server `Bots test (802975314103238656)`:
>  `local-.pattern`
> `local-.pattern2`